### PR TITLE
APP-3133: Add follow/unfollowUser api to UserService

### PR DESCRIPTION
**Pull request by symphony-hong**
### Ticket
[APP-3133](https://perzoinc.atlassian.net/browse/APP-3133)

### Description
Add follow/unfollowUser api to UserService

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [x] Unit tests updated or added
- [x] Javadoc added or updated

### CODE REVIEW

**Review by symphony-elias — APPROVED** [review]
LGTM!

File: symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java [hunk 1]
```diff
@@ -96,4 +96,24 @@
...
+   */
+  void followUser(@Nonnull Long uid, @Nonnull List<Long> followerIds);
+
+  /**
+   * Make a list of users to stop following a specific user.
+   * {@link UserService#unfollowUser(Long, List)}
+   *
+   * @param uid         The id of the user to be unfollowed.
+   * @param followerIds List of the ids of the followers.
+   * @see <a href=https://developers.symphony.com/restapi/v20.9/reference#unfollow-user">Unfollow User</a>
```
Comment (symphony-thibault) [inline symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java:116]: You missed the first double quote in `href`

Comment (symphony-hong) [reply symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java:116]: Eagle eyes! 🦅

**Review by symphony-thibault — APPROVED** [review]
LGTM 👍

**Pull request merged**